crates/mun_language_server/src/state.rs

```diff
@@ -375,6 +377,23 @@ impl LanguageServerSnapshot {
 
         Ok(url)
     }
+
+    /// Converts the specified `Url` to a `hir::FileId`
```

Comment:
I see url and uri used interchangeably. Why's that?

Reply:
Its because LSP does the same. A text document is referred to by (copied from lsp_types):
```
/// The text document's URI.
pub uri: Url,
```